src/main/java/com/chess/backend/gamemodel/constants/Color.java

```diff
@@ -1,14 +1,16 @@
 package com.chess.backend.gamemodel.constants;
 
 public enum Color {
-    WHITE("White"),
-    BLACK("Black"),
-    RED("Red"),
-    BLUE("Blue");
+    WHITE("White", 0),
```

Comment:
theoretically you don't need the position, you can also use this form Color.values()[0] to get White for example